Controls/Mail/Mailman.c

```diff
@@ -21,7 +33,20 @@ void depositLetter(void) {
  */
 void Task_Mailman(void* p_arg) {
 	OS_ERR err;	// Make sure to check for errors and print the error code if not OS_ERR_NONE
+	int i = 0;
+	char letter = 'a';
+
 	while(1){
+		depositLetter(i, letter);
+		OSTimeDlyHMSM(0,0,3,0,OS_OPT_TIME_HMSM_STRICT,&err);
+		if (err != OS_ERR_NONE){
+			printf("Error %d\n", err);
+			return;
+		}
 
+		if(i < 255){
+			i++;
+			letter++;
+		}
```

Comment:
You might want to add a condition here that resets the string when it reaches maximum capacity so you don't overflow the string when i becomes more than the total length. Technically this is only unique until it reaches the last position in the string.